site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -162,23 +174,28 @@ If the installation is in `production` mode, apply the migrations out of band.
 
 When running in development runtime mode, silent mode was enabled based upon the presence of environment variables, such as the database user, and could not explicitly be enabled or disabled. 
 
-=== Docker tags
+=== Docker Tags
 
-The docker compose file references the `latest` tag, but that tag is not dynamic, it is only the latest at a point in time. To get the most recently released image, you have a couple of options.
+The docker compose file references the `latest` tag, but that tag is not dynamic. It is only the latest at a point in time. To get the most recently released image, you have a couple of options.
 
-* You can edit the docker compose file with an explicit tag version. This is a good idea for a production deployment.
-* You can remove the old image first, as otherwise the latest image won't be used: `docker rmi <old image id>`. This command may prompt you to remove containers using that image. Since all state is stored in the database, you can safely remove the containers.
-* You can pull the latest image with this command: `docker pull fusionauth/fusionauth-app:latest`.
+* Edit the docker compose file and specify a specific version. This is a good idea for a production deployment.
+* Remove the old image first: `docker rmi <old image id>`. This command may prompt you to remove containers using that image. Since all state is stored in the database, you can safely remove the containers. Then, when you restart docker compose, the latest image will be retrieved.
+* Pull the latest image with this command: `docker pull fusionauth/fusionauth-app:latest`.
 
-== Docker Images
+== Custom Docker Images
 
-If you want to build your own image starting with our base image, the following Docker image is available.
+If you want to build your own image starting with our base image, start with the `fusionauth/fusionauth-app` image. Just as the FusionAuth docker image is based on the `ubuntu:focal` image, you can build a docker file which is based on the `fusionauth:fusionauth-app` image. This can be useful if you want to automatically install a password hashing plugin, configuration file, or other customization.
 
-=== FusionAuth App
+Here's the FusionAuth application Docker image file as a reference.  
+
+[source,yaml,title=The FusionAuth Docker file]
+----
+include::https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/fusionauth-app/Dockerfile[]
+----
 
-```
-docker pull fusionauth/fusionauth-app
-```
+FusionAuth supports multiple architectures which may not be the case for your needs. There is a build step and then an image creation step.
```

Comment:
I don't quite follow. I think this is saying that FusionAuth supports multiple architectures, but the supported architectures may not meet the customer's needs? Is the reference to a "build step" and "image creation step" meant to provide a hint as to where a customer could customize to fit their architecture?

I think that "may not be the case" doesn't quite fit here. It's always the case that FusionAuth supports multiple architectures. I'm not sure what a replacement should be without understanding the context here.